pyaerocom/aeroval/glob_defaults.py

```diff
@@ -222,7 +222,61 @@
     "colmap": "coolwarm",
     "unit": "1",
     "decimals": 2
-  }
+  },
```

Comment:
I trust that these changes are coordinated with @AugustinMortier

Reply:
Yes, I added this at his request.